lua/telescope/_extensions/file_browser/utils.lua

```diff
@@ -118,6 +119,8 @@ fb_utils.redraw_border_title = function(current_picker)
     else
       new_title = finder.cwd
     end
+    local width = vim.api.nvim_win_get_width(0) * 0.80
```

Comment:
Slight nit. `0` here refers to the prompt window rather than the results window. In, for instance, `ivy` theme, they are of different width. Not sure if they otherwise can differ but let's refer to the correct window just in case like

```lua
local width = math.floor(a.nvim_win_get_width(current_picker.results_win) * 0.8)
```

should do the job as intended.